integration-tests/hibernate-orm-panache/jackson-integration/src/test/java/io/quarkus/it/panache/TestEndpointTest.java

```diff
@@ -0,0 +1,26 @@
+package io.quarkus.it.panache;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.h2.H2DatabaseTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+@QuarkusTestResource(H2DatabaseTestResource.class)
+public class TestEndpointTest {
+
+    @Test
+    public void testHelloEndpoint_noPersistantAttributeIsThere() {
+        given()
+                .when().get("/test") //
+                .then() //
+                .statusCode(200) //
+                // here is checked that the property 'persistent' is missing
+                .body(is("{\"id\":null,\"name\":\"max\"}"));
```

Comment:
This doesn't seem right: the body should be `{"name": "max"}`.

Reply:
> Hi. Thanks for the PR. The fix looks good, but I don't think we should add a new integration test just for this. Can't we add a unit test to the existing integration test using Jackson explicitely instead?

Hi,
thanks for the review!
I have build a new integration test module for this because I do not know how to configure Quarkus to use JSONB and Jackson together... I was following this guide: https://quarkus.io/guides/rest-json 
How can I tell Quarkus to use JSONB or Jackson for a resource or method explicitly?

> This doesn't seem right: the body should be `{"name": "max"}`.

Yes, this is not right.
The behaviour with jsonb is different. There it is `{"name": "max"}` with no `"id":null`.
In PanacheEntity is mentioned that the accessors for id are generated but I do not know how...
I do not know to fix this. Do you have a hint?

Reply:
Don't use REST, just write a unit test that serialises using the Jackson API?

Apologies, my bad, I misread and I think it's right that if the `id` is null it can be serialised as such. I initially thought it was `persistent` instead of `id`. Just ignore this :)

Reply:
> Don't use REST, just write a unit test that serialises using the Jackson API?

Thank you for that hint. Did it.